src/org/scada_lts/web/mvc/api/EventsAPI.java

```diff
@@ -95,6 +100,54 @@
         }
     }
 
+    /**
+     * Acknowledge specific Event Alarm from REST API
+     *
+     * @param eventId Event ID number
+     * @param request Request containing user data
+     * @return Response
+     */
+    @PutMapping(value = "/silence/{id}")
+    public ResponseEntity<String> silenceEvent(@PathVariable("id") int eventId, HttpServletRequest request) {
+        LOG.info("PUT::/api/events/silence/" + eventId);
+        try {
+            User user = Common.getUser(request);
+            if (user != null) {
+                Date time = new Date();
+                eventService.silenceEvent(eventId, time.getTime(), user.getId(), 0);
+                return new ResponseEntity<>(HttpStatus.OK);
+            } else {
+                return new ResponseEntity<>(HttpStatus.UNAUTHORIZED);
+            }
+        } catch (Exception e) {
+            return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
+        }
+    }
+
+    /**
+     * Acknowledge specific Event Alarm from REST API
+     *
+     * @param eventId Event ID number
+     * @param request Request containing user data
+     * @return Response
+     */
+    @PutMapping(value = "/unsilence/{id}")
+    public ResponseEntity<String> unsilenceEvent(@PathVariable("id") int eventId, HttpServletRequest request) {
+        LOG.info("PUT::/api/events/unsilence/" + eventId);
+        try {
+            User user = Common.getUser(request);
+            if (user != null) {
+                Date time = new Date();
+                eventService.unsilenceEvent(eventId, time.getTime(), user.getId(), 0);
```

Comment:
Remove` Date time` and unnecessary arguments ->
`eventService.unsilenceEvent(eventId, user.getId());`

Reply:
git push